src/mono/mono/mini/interp/transform.h

```diff
@@ -147,11 +147,12 @@ struct _InterpCallInfo {
 	// in the order they are pushed to the stack. This makes it easy to find
 	// all source vars for these types of opcodes. This is terminated with -1.
 	int *call_args;
+	int call_offset;
```

Comment:
`call_end_offset` has become temporary variable needed only during the offset allocation and it might be removed from the `call_info` struct.

Reply:
We added the `call_end_offset` field so we can easily map between call `InterpInst` to allocated offset, without use of other data structures and lookups. Did you have an alternative for this ?

Reply:
Without other data structures, no. It can be inferred from `base_offset` + `call_args` but it is probably inefficient.